projects/deno.land/package.yml

```diff
@@ -17,8 +17,28 @@ runtime:
   env:
     DENO_NO_UPDATE_CHECK: 'true'
 
+companions:
+  # denort added in v1.41.0 requires `unzip`
+  info-zip.org/unzip: '*'
+
+dependencies:
+  darwin/x86-64:
+    llvm.org: 17 # libunwind
+
 build:
   script:
+    # https://github.com/denoland/deno/pull/22298
+    # deno.land 1.41.0 will currently _not_ run deno compile on linux/aarch64
+    # for their first official release, they're using ubuntu 22.04, which means
+    # a newer glibc. Patching via the https://github.com/LukeChannings/deno-arm64
+    # repo may be possible, but lets not delay the three usable arches for the rare
+    # one. Revist this.
+
+    # Obviously, I'd just skip the test, except everything we _do_ relies on
+    # deno compile, so that'd just break the unholy everything out of pkgx.
+    - run: test "{{hw.platform}}+{{ hw.arch }}" != "linux+aarch64"
+      if: 1.41.0
+
```

Comment:
@mxcl

Reply:
Could screw up our own pipelines, I guess we’ll find out! But yeah good call.